Packs/PAN-OS/Integrations/Panorama/Panorama_test.py

```diff
@@ -161,6 +161,43 @@ def test_filter_rules_by_status(disabled: str, rules_file: str, expected_results
     assert result == expected_result
 
 
+def test_get_address(mocker):
+    """
+    Given:
+     - an address_name argument which does not exist
+
+    When:
+     - running the panorama_get_address function
+
+    Then:
+     - Ensure the return value is an empty dictionary
+    """
+    import Panorama
+    from Panorama import panorama_get_address
+    exception_msg = 'Object was not found, verify that the name is correct and that the instance was committed.'
+    mocker.patch.object(Panorama, "http_request", side_effect=Exception(exception_msg))
+    result = panorama_get_address("lksdhgkjhdsga")
+    assert result == {}
```

Comment:
please check that `return_results` was called with the appropriate message

Reply:
Here there is not return_results, just a return value (dict)  from 'panorama_get_address' func